app/src/main/java/ch/epfl/sdp/database/room/AppDatabase.java

```diff
@@ -12,20 +12,26 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 
-@Database(entities = {LeaderboardEntity.class}, version = 1)
+@Database(entities = {GeneralLeaderBoardEntity.class}, version = 2, exportSchema = false)
```

Comment:
What does this change?

Reply:
It seems I change the name of the table, then I need to upgrade the version.